cdbot/constants.py

```diff
@@ -113,7 +113,7 @@ class Exchange:
 ROOT_ROLE_ID = int(environ.get("ROOT_MEMBERS_ID", "450113490590629888"))
 SUDO_ROLE_ID = int(environ.get("SUDO_MEMBERS_ID", "450113682542952451"))
 ADMIN_ROLES = ("Root", "Sudo")
-BANNED_DOMAINS = ["discord.gg"]
```

Comment:
I don't think we actually ever call this.

Reply:
true

Reply:
I changed it, checked for usage and figured there's little point changing it back

Reply:
Yeah, may as well